### PR TITLE
added hello world

### PR DESCRIPTION
**Pull request by TDGLU**
Added "Hello World"

### CODE REVIEW

**Review by Urstupidsammy — COMMENTED** [review]
thats cool -LUKE

**Review by Urstupidsammy — APPROVED** [review]
im on it

**Comment by TDGLU** [comment]
> thats cool -LUKE

Thanks!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # groupProject
+
 I like using Javascript to make my webpages more interactive.
 
+HELLO WORLD
```
Comment (TDGLU) [inline README.md:5]: Comment

**Pull request merged**